src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaControllerTest.java

```diff
@@ -30,6 +30,8 @@
 
 @Category(GradleTaskForRca.class)
 public class RcaControllerTest {
+  private static final int IEVAL_ITERATIONS = 20;
+  private static final int NODE_ROLE_EVAL_ITERATIONS = 70;
```

Comment:
Is it still required after merging with the last changes ? It never failed for me. If we think it is required, then we should add a comment explaining why this is so

Reply:
It is required as we are changing the periodicity of getting master node value to 60 sec. Will add a comment for that